cernopendata/modules/fixtures/data/docs/cms-guide-docker/cms-guide-docker.md

```diff
@@ -27,29 +26,27 @@ Digest: sha256:6b9a12992ba088a168b87df98a841d3c56dede326684f5551368fd359acfb43c
 Status: Downloaded newer image for clelange/cmssw:5_3_32
 Setting up CMSSW_5_3_32
 CMSSW should now be available.
-[15:10:10] cmsusr@eb9ecf54fd2a ~/CMSSW_5_3_32/src $
 ```
 
-Once done, you should see the commmand prompt (`docker run -i` means run interactively):
+Once done, you should see the commmand prompt for the CMSSW instance within Docker:
 
-`
+```bash
 cmsusr@eb9ecf54fd2a ~/CMSSW_5_3_32/src $
-`
+```
 
 ### Instructions for 2010 data
 
 Once Docker is installed, you can fetch a CMSSW image and create and start a container using the `docker run` command:
 
-`
+```bash
```

Comment:
If you put the prompt, i.e. the block is a shell session example, then it had better be `console`.  If the block contains `shell` or `bash` script code, then it should not contain prompt, as it should be copy-paste-runnable.

Examples: 

```
'''console
$ whoami
$ date
'''

'''sh
whoami
date
'''
```

Note that the former can contain the command outputs:

```
'''console
$ whoami
root
$ date
Tue 25 Jun 16:16:50 CEST 2019
'''
```

while the second (`sh`) cannot.

(Note that this remark also applies to other blocks in this PR.)

Reply:
Ugh, I thought I had removed this everywhere. Thanks for the pointers, I'll fix it everywhere.